### DISCUSSION TITLE
Account hacked github support is rubbish ,a week no reply from support

### DISCUSSION THREAD

**Discussion by ghost** [community/community#111095] [New to GitHub] [answered]
my account got hacked and username is no longer accessable,neither my email id is not linked ,.so the itself i raised a support ticket from this new account,but no reply from support team yet.are they ever reply ? what the rubbish service.anyone has any clue how to get response from these people.my main account gone all my projects and all my codes are gone.help its urgent.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @pk397sad, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
